include/biosHandler.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+#include <sdbusplus/asio/connection.hpp>
+#include <sdbusplus/bus.hpp>
+
+#include <iostream>
+
+namespace vpd
+{
+/**
+ * @brief A class to operate upon and back up some of the BIOS attributes.
+ *
+ * The class initiates call backs to listen to PLDM service and changes in some
```

Comment:
its not clear where BIOS attributed are saved to apply the changes.

Reply:
> its not clear where BIOS attributed are saved


It is explained in the last line of description.

Reply:
okay